cargo-concordium/src/build.rs

```diff
@@ -1202,6 +1206,13 @@ pub(crate) fn build_and_run_integration_tests(
 
         command.env("CARGO_CONCORDIUM_TEST_ALLOW_DEBUG", "1");
     }
+    // This enviroment variable needs to match the
+    // `CONTRACT_MODULE_OUTPUT_PATH_ENV_VAR` constant in the `contract-testing`
+    // crate.
+    command.env(
+        "CARGO_CONCORDIUM_TEST_MODULE_OUTPUT_PATH",
+        build_info.out_filename,
```

Comment:
**Question:** Can we be sure this will always be an absolute path?

Reply:
Why does it matter if it is absolute or relative?

Reply:
It might not matter, but I would be concerned that using a relative path would not work for every setup.

Reply:
I don't think it should matter, since the `cargo test` process is spawned by `cargo-concordium`, so they should agree on paths in this case. I guess if we wanted to use the environment variable in other contexts it might matter, though.